eng/pipelines/eng-common-sync.yml

```diff
@@ -48,29 +80,76 @@ stages:
               PRBody: >
                 Sync ${{ parameters.DirectoryToSync }} directory with azure-sdk-tools repository.<br>
                 See [eng/common workflow](https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/README.md#workflow)
-            PRDataArtifactPath: $(Build.ArtifactStagingDirectory)/${{ parameters.PRDataFileName }}
-            Repos:
-              - azure-sdk-for-android
-              - azure-sdk-for-c
-              - azure-sdk-for-cpp
-              - azure-sdk-for-go
-              - azure-sdk-for-ios
-              - azure-sdk-for-java
-              - azure-sdk-for-js
-              - azure-sdk-for-net
-              - azure-sdk-for-python
-
-        - task: PublishPipelineArtifact@1
-          condition: succeeded()
-          inputs:
-            artifactName: ${{ parameters.ArtifactName }}
-            path: $(Build.ArtifactStagingDirectory)/${{ parameters.PRDataFileName }}
+            Repos: ${{ parameters.Repos }}
+
+  - stage: CreateSyncPRs
+    jobs:
+      - deployment: CreateSyncPullRequests
+        displayName: Create Sync Pull Requests
+        environment: github
```

Comment:
Probably should use githubmerges here as well.

Reply:
We still want to wait for the PR to be approved before running the VerifyAndMerge stage.

Reply:
Yes but you can use the environment on each job and they are each separate approvals. The key is we want an environment that doesn't time out in 3 days which is what githubmerges gives us 30 days instead of the 3 days github gives us.